docs/man_pages/lib-management/plugin-find.md

```diff
@@ -0,0 +1,26 @@
+plugin find
+==========
+
+Usage | Synopsis
+---|---
+Finds NativeScript plugins in npm | `$ tns plugin find [<PluginName>] [--all] [--count <Count>]`
+
+Finds NativeScript plugins in npm.
+
+### Options
+* `--all` - Specifies that all results will be displayed at once.
+* `--count` - Specifies the count of plugins to display.
```

Comment:
Somewhere you need to note that --all and --count are mutually exclusive. 

"You cannot set `--all` and `--count` simultaneously."